software/core/Infrastructure/AngularFrontend/src/app/components/dashboard/speed/speed.component.html

```diff
@@ -1,4 +1,22 @@
+<style>
+    .text{
+        position: absolute;
+        z-index: 999;
+        margin: 0 auto;
+        left: 0;
+        right: 0;
+        top: 35%;
+        text-align: right;
+        width: 20%;
+        font-size: 0.50rem;
+        color: red;
+    }
+</style>
 <div style="text-align: center;">
+    <div [hidden]="!engaged" class="text">
```

Comment:
I am not sure if we should just go with class name "text" (compiled Angular might scream at us). Maybe add a another class name, maybe "brake" and in css call ".text.brake"

Reply:
Sorry, I copied in pasted the CSS and did not change the name. Fixed now.